packages/component/src/Activity/Bubble.js

```diff
@@ -50,7 +50,11 @@ Bubble.propTypes = {
   fromUser: PropTypes.bool,
   nub: PropTypes.bool,
   styleSet: PropTypes.shape({
-    bubble: PropTypes.any.isRequired
+    bubble: PropTypes.any.isRequired,
+    options: PropTypes.shape({
+      bubbleNubSize: PropTypes.number.isRequired,
+      bubbleFromUserNubSize: PropTypes.number.isRequired
+    })
```

Comment:
This is nitpicking.

The `options` should appears in `styleSet` because our code internally do that, and there should be no way we missed the `options`. Thus, having `isRequired` give a thin layer of protection.

```suggestion
    }).isRequired
```